content/sensu-core/2.0/reference/entities.md

```diff
@@ -14,21 +14,21 @@ menu:
 
 ### Why "Entities"?
 
-Sensu 2.0 uses an updated data model to address the latest paradigm shift in infrastructure and software delivery like containers and "serverless". Sensu is no longer confined to the concept of a client, we have "entities" that can can represent anything.
+Sensu 2.0 uses an updated data model to address the latest paradigm shift in infrastructure and software delivery like containers and "serverless", so we've adjusted the nomenclature accordingly. Sensu is no longer confined to the concept of a client, we have "entities" that can can represent anything.
```

Comment:
I feel like we can get even more direct by acknowledging the need for new "thing" to be monitored. Maybe: 

Sensu 2.0 uses an updated data model that allows for it to acknowledge the full range of infrastructure, runtime and application types that compose a complete monitoring environment (from server hardware to "serverless" functions). Sensu no longer focuses on the object doing the monitoring (previously called a "client") and instead focuses on the object it monitors. We call these monitored parts of an infrastructure "entities."